packages/babel-preset-fbjs/plugins/inline-requires.js

```diff
@@ -69,6 +69,13 @@ module.exports = babel => ({
                 return;
               }
               const {declarationPath, moduleName} = parseResult;
+              const requireName = path.node.callee.name || path.node.callee.property.name;
+
+              // If declaration path has require binding in the scope then
+              // don't inline calls
+              if(declarationPath.scope.getBinding(requireName) != null) {
+                return;
+              }
```

Comment:
I suggest to move the logic for getting the `requireName` and testing if there's no override in the current or any parent scope into the `getInlineableModule` function

```
function getInlineableModule(path, state) {
  const node = path.node;
  const fnName = node.callee.name;

  const isInlineable =
    node.type === 'CallExpression' &&
    node.callee.type === 'Identifier' &&
    state.inlineableCalls.has(fnName) &&
    node['arguments'].length >= 1 && 
    path.scope.getBinding(fnName) == null;

  if (!isInlineable) {
    return null;
  }
  
  // ....

  return moduleName == null || state.ignoredRequires.has(moduleName)
    ? null
    : { moduleName, requireFnName: fnName };
}
```

You'll have to rewrite the two call-sites to pass a `path` instead of a `node` and to get the `moduleName` and `requireFnName` from the returned object.

Reply:
Done. Can you please check the updated code